config/vufind/config.ini

```diff
@@ -840,6 +847,14 @@ authors         = Wikipedia
 url = "https://api.vlb.de/api/v1/cover/"
 ; token = "XXXXXX-XXXX-XXXXX-XXXXXXXXXXXX"
 
+; This section is needed for LocalFile cover loading.
+; Note:  All coverimage LocalFile:ConfigSection must include the following keys: 
+;        filepath, filename, and imagetype.
+[LocalFile]
+filepath = /usr/local/vufind/local/coverimages
+filename = recordid
```

Comment:
Yes, that makes a lot of sense... though if we go that far, do we need a config section for this at all, or should we just put the whole path after the colon in the cover setting itself, so you end up with `LocalFile:/path/to/file-pattern`? The only downside to that I can see is that it might complicate the parsing a little bit, but I don't think it would be a major obstacle.